code/go/0chain.net/conductor/conductor/executor.go

```diff
@@ -303,20 +303,31 @@ func (r *Runner) WaitShareSignsOrShares(ssos config.WaitShareSignsOrShares,
 func (r *Runner) WaitAdd(wadd config.WaitAdd, tm time.Duration) (err error) {
 
 	if r.verbose {
-		log.Printf(" [INF] wait add miners: %s, sharders: %s, blobbers: %s",
-			wadd.Miners, wadd.Sharders, wadd.Blobbers)
+		log.Printf(" [INF] wait add miners: %s, sharders: %s, blobbers: %s, authorizers %s",
+			wadd.Miners, wadd.Sharders, wadd.Blobbers, wadd.Authorizers)
 	}
 
 	r.setupTimeout(tm)
 	r.waitAdd = wadd
 	if wadd.Start {
 		// start nodes that haven't been started yet
-		for _, name := range append(wadd.Sharders, append(wadd.Miners, wadd.Blobbers...)...) {
+		allNodes := append(wadd.Sharders, wadd.Miners...)
+		allNodes = append(allNodes, wadd.Blobbers...)
+		allNodes = append(allNodes, wadd.Authorizers...)
+
+		for _, name := range allNodes {
 			if err := r.doStart(name, false, false); err != nil {
 				return err
 			}
 		}
 	}
+
+	// filter initialized nodes
+	nodes := r.server.Nodes()
+	for name, _ := range nodes {
```

Comment:
please do `for name := range nodes {
}` to fix lint error

Reply:
fixed by [683f64a](https://github.com/0chain/0chain/pull/1799/commits/683f64a9a0f94ad944c5dda65996c1e747a74f90). Thanks for providing the solution